### PR TITLE
Feature | EKS | Add ArgoCD notifcations and kube prometheus stack

### PR DESCRIPTION
**Pull request by angelofenoglio**
## What?
* Add Notifications to ArgoCD
* Add Kube Prometheus Stack to cluster components, make sure the data is persisted
* Make ArgoCD and Kube Prometheus Stack run on their own managed node groups

### CODE REVIEW

**Comment by angelofenoglio** [comment]
- Initially the PVCs were meant to be created using AWS EBS CSI driver but this required adding addons management to this cluster which would end up complicating this stack.
- Another thing to take into consideration is the fact that EBS volumes can only exist in one AZ, but spot instances can be assigned from any AZ which sometimes makes these volumes unreachable from the prometheus/grafana/alertmanager pods perspective. An implementation of EFS volumes would be needed to ensure complete reliability. However, these volumes cannot be allocated dynamically and so need to be created beforehand.

File: apps-devstg/us-east-1/k8s-eks/identities/ids_fluentbit.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  description = "Fluent Bit"
+  tags        = local.tags_fluent_bit
+  policy      = <<-EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "es:*"
```
Comment (diego-ojeda-binbash) [inline apps-devstg/us-east-1/k8s-eks/identities/ids_fluentbit.tf:38]: @angelofenoglio Could you add a TODO note here regarding this overly permissive access?

**Review by diego-ojeda-binbash — DISMISSED** [review]
Pre-approved with a minor comment.

File: apps-devstg/us-east-1/k8s-eks/identities/ids_fluentbit.tf (hunk 1 above)
Comment (angelofenoglio) [reply apps-devstg/us-east-1/k8s-eks/identities/ids_fluentbit.tf:38]: Done!

**Comment by angelofenoglio** [comment]
I think we should replicate this in the demoapps cluster, What do you think @exequielrafaela @diego-ojeda-binbash?

**Review by exequielrafaela — APPROVED** [review]

**Pull request merged**